Sources/MapboxMaps/Viewport/ViewportStatusObserver.swift

```diff
@@ -0,0 +1,19 @@
+public protocol ViewportStatusObserver: AnyObject {
+    func viewportStatusDidChange(from fromStatus: ViewportStatus,
```

Comment:
this is named as `onViewportStatusChanged` on Android as it feels more natural to match previous naming.